sdks/python/apache_beam/utils/timestamp_test.py

```diff
@@ -132,9 +132,9 @@ def test_arithmetic(self):
     with self.assertRaises(TypeError):
       self.assertEqual(Duration(456) - Timestamp(123), 333)
     with self.assertRaises(TypeError):
-      self.assertEqual(-Timestamp(123), -123)
+      self.assertEqual(-Timestamp(123), -123)  # pylint: disable=invalid-unary-operand-type
     with self.assertRaises(TypeError):
-      self.assertEqual(-Timestamp(123), -Duration(123))
+      self.assertEqual(-Timestamp(123), -Duration(123))  # pylint: disable=invalid-unary-operand-type
```

Comment:
Thanks, this looks better. This test is specifically checking unsupported operations so it makes sense that we'd need to disable a pylint check.